app/services/doc_auth/response.rb

```diff
@@ -1,6 +1,7 @@
 module DocAuth
   class Response
-    attr_reader :errors, :exception, :extra, :pii_from_doc
+    attr_reader :errors, :exception, :pii_from_doc
+    attr_accessor :extra # so we can chain extra analytics
```

Comment:
I'm not sure I like this `attr_accessor` ... I like the idea of immutable response objects --- if we want to combine stuff in, how about we use `#merge` to combine in new ones? 

Also techincally since `extra` is a hash, which is mutable by default, we could modify the `extra` with just an `attr_reader` still (not that I would recommend it)

Reply:
yeah wasn't crazy about it when i wrote it.  i thought about just recreating the objects.  i'll go down the merge path...